cmd/cluster/create.go

```diff
@@ -92,6 +92,7 @@ func NewCreateCommand() *cobra.Command {
 
 	cmd.MarkFlagRequired("pull-secret")
 	cmd.MarkFlagRequired("aws-creds")
+	cmd.MarkFlagRequired("base-domain")
```

Comment:
If we call `create infra` first and just pass the infra.json to `create cluster` is the `base-domain` still required?